### PR TITLE
Enhance input json files

### PR DESCRIPTION
**Pull request by YunziZhangAzure**
## Problem
<Please describe what problem this PR is trying to resolve>
Current input json is not complete

## Solution
<Please elaborate the solution for the problem>
1. Add some missing fields in input json
2. Since name of vnet is required, test pipeline will use jq to populate this field dynamically.

## Tests
<Please provide steps to test the PR>
See test in test pipeline

## Notes
<Additional comments for the PR>

### CODE REVIEW

File: deploy/terraform/bootstrap/sap_deployer/deployer.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "environment": "np",
     "vnets": {
       "management": {
+        "name": "NP-EAUS-DEP00-vnet",
```
Comment (rukawata) [inline deploy/terraform/bootstrap/sap_deployer/deployer.json:7]: vnet name would follow naming convention, it makes sense to have it in deployer_full.json but not the deployer.json. Any specific reasons?

**Review by rukawata — COMMENTED** [review]
Thanks for adding the missing parameters :) Just question regarding one input param.

File: deploy/terraform/bootstrap/sap_deployer/deployer.json (hunk 1 above)
Comment (YunziZhangAzure) [reply deploy/terraform/bootstrap/sap_deployer/deployer.json:7]: It seems this field is required according to the configuration doc from PM.
https://github.com/Azure/sap-hana/blob/documentation/v2.3/documentation/SAP_Automation_on_Azure/Software_Documentation/configuration-deployer.md

Comment (rukawata) [reply deploy/terraform/bootstrap/sap_deployer/deployer.json:7]: I see! Thanks for going through the documentation.
I think this is incorrect in the documentation, naming convention should be used when possible :)

**Pull request closed without merging**